### ISSUE TITLE
Sponsor block level labels have insufficient contrast

### ISSUE THREAD

**Issue by jdleonard** [Drupal-NYC/drupalcampnyc#83]
For example, "Times Square (Venue) sponsor" is a light grey on a white background.

**Comment by ccaajj** [comment]
had a merge conflict, @jdleonard you changed opacity to 0.7? That was the conflict so I went with the 0.7 instead of 1.

**Comment by jdleonard** [comment]
I did. Let me know if you need any assistance.

**Comment by ccaajj** [comment]
fixed #83 reconciled and kept the 0.7 value

**Comment by AmyJuneH** [comment]
https://www.drupal.org/project/drupalnyc/issues/3182841
